### ISSUE TITLE
Visualize dispersion

### ISSUE THREAD

**Issue by fkohrt** [visa/visa-chart-components#69]
I love how the Visa Chart Components incorporate best practices for accessibility, e.g. always provide a table representation of the data. But as soon as the components are used for inference rather than description, VCC lacks a good way to visualize the dispersion of data, e.g. via confidence intervals around point estimates. This could be implemented as error bars around bar charts or bands/ribbons around line charts.

Providing information about the variance / standard error besides a variable's mean is considered best practice in the communication of statistical results and I would love to see that implemented for VCC.
